security/advancedtls/advancedtls.go

```diff
@@ -415,12 +415,12 @@ func (o *ServerOptions) config() (*tls.Config, error) {
 // advancedTLSCreds is the credentials required for authenticating a connection
 // using TLS.
 type advancedTLSCreds struct {
-	config           *tls.Config
-	verifyFunc       CustomVerificationFunc
-	getRootCAs       func(params *GetRootCAsParams) (*GetRootCAsResults, error)
-	isClient         bool
-	verificationType VerificationType
```

Comment:
Any reason why you changed the order here (and across the test)?

Reply:
Copy/paste changes, I can change it to retain the ordering if you think it's worth it

Reply:
Oh ok I thought you wanted to sort it alphabetically